pylint/extensions/_check_docs_utils.py

```diff
@@ -58,10 +58,19 @@ def possible_exc_types(node):
     """
     excs = []
     if isinstance(node.exc, astroid.Name):
-        excs = [node.exc.name]
+        inferred = safe_infer(node.exc)
+        if inferred:
```

Comment:
Try to exclude astroid.Uninferable, which can be returned by safe_infer. Maybe we should make it have a false value by default.